docs/products/opensearch/reference/sunsetting-backward-compatible-aiven-for-opensearch.rst

```diff
@@ -0,0 +1,163 @@
+Sunsetting backward compatibility with Aiven for Elasticsearch
+==============================================================
+
+In September 2021, Aiven `introduced Aiven for OpenSearch® <https://aiven.io/blog/announcing-aiven-for-opensearch>`_ service that is backward compatible with Aiven for Elasticsearch.
```

Comment:
"that is backward compatible" in present tense will not age nicely after August 23rd. I'm not sure how to rephrase it, but I think it is worth changing the sentence, so it is valid after August 23rd